kong/plugins/zipkin/schema.lua

```diff
@@ -62,6 +62,9 @@ return {
           { static_tags = { type = "array", elements = static_tag,
                             custom_validator = validate_static_tags } },
           { http_span_name = { type = "string", required = true, default = "method", one_of = { "method", "method_path" } } },
+          { connect_timeout = {type = "number", default = 5000 } },
+          { send_timeout = {type = "number", default = 10000 } },
+          { read_timeout = {type = "number", default = 10000 } },
```

Comment:
Some of our plugins use `2000` as the default timeout value, and considering the data size, the send_timeout and read_timeout should be larger than the default value; I think `5000` should be enough; what do you think?
```suggestion
          { connect_timeout = typedefs.timeout { default = 2000 } },
          { send_timeout = typedefs.timeout { default = 5000 } },
          { read_timeout = typedefs.timeout { default = 5000 } },
```

Reply:
Works for me!